src/views/home/components/report-issue-button/report-issue-button.styles.js

```diff
@@ -9,11 +9,13 @@ const reportIssueButtonStyles = createUseStyles((theme) => ({
     background: theme.palette.white
   },
   text: {
-    width: '100%',
-    textAlign: 'center',
     fontWeight: theme.fontWeights.medium,
     color: theme.palette.grey.main,
     padding: `${theme.spacing(1.5)}px 0`
+  },
+  separator: {
+    color: theme.palette.grey.main,
+    padding: theme.spacing(1.5)
```

Comment:
I think that the separator only needs to have padding on the sides right? Like: `${theme.spacing(1.5)}px 0`

Reply:
Nope it needs both:

<img width="353" alt="Screenshot 2021-04-29 at 15 34 17" src="https://user-images.githubusercontent.com/2448210/116559131-65ec2100-a900-11eb-84a5-ec8e405d9138.png">
<img width="393" alt="Screenshot 2021-04-29 at 15 34 23" src="https://user-images.githubusercontent.com/2448210/116559135-671d4e00-a900-11eb-9054-25746ad534a0.png">